### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by awolfe76**
Still getting `403` on the API calls, but this handles that response better. :)

Also, does a check to see if authenticated and renders a quick heading if not ... which we can make better sometime soon.

### CODE REVIEW

File: src/Alert.css [hunk 1]
```diff
@@ -99,5 +99,5 @@
 }
 
 .alert-paragraph {
-  width: 66ch;
+  max-width: 66ch;
```
Comment (awolfe76) [inline src/Alert.css]: This lets the error on delete render a bit cleaner because it's in a `<table>`.

File: src/App.js [hunk 2]
```diff
@@ -21,38 +21,62 @@ class App extends Component {
...
-              </button>
-            </nav>
-          </header>
-          <Route exact path="/" component={Search} />
-          <Route exact path="/add" component={Institution} />
-          <Route exact path="/update" component={Institution} />
-        </div>
-      </Switch>
-    )
+    const ProtectedRoute = ({ component: Component, ...rest }) => {
```
Comment (awolfe76) [inline src/App.js]: This lets us pass the `token` prop. Seems like a weird way to do it, but kind of neat too.

File: src/institution/index.js [hunk 3]
```diff
@@ -85,18 +86,29 @@ class Institution extends Component {
     fetch(`/v2/admin/institutions`, {
       method: method,
       body: JSON.stringify(institution),
-      headers: { 'Content-Type': 'application/json' }
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: 'Bearer ' + token
```
Comment (awolfe76) [inline src/institution/index.js]: This seems right, am I missing something that would cause the `403`?

File: src/institution/index.js [hunk 4]
```diff
@@ -120,96 +132,103 @@ class Institution extends Component {
           : 'If any data fields other than Respondent Name or Email Domain need to be updated, please escalate the case to Tier 2 for further support.'
     }
 
-    if (pathname === '/update' && !state) return <h1>NOOOOO!</h1>
+    if (pathname === '/update' && !state) this.props.history.push('/')
```
Comment (awolfe76) [inline src/institution/index.js]: Just send that back to the homepage if something went completely wrong. We can add `state` to this to show some kind of message on the homepage.

File: src/institution/index.js (hunk 3 above)
Comment (wpears) [reply src/institution/index.js]: Nope, the auth header is correct. Looks like a backend issue to me.

File: src/App.js (hunk 2 above)
Comment (wpears) [reply src/App.js]: :+1:

**Pull request merged**